test/expected/create_hypertable.out

```diff
@@ -1,6 +1,6 @@
 \c single :ROLE_SUPERUSER
 create schema test_schema AUTHORIZATION :ROLE_DEFAULT_PERM_USER;
-\c single :ROLE_DEFAULT_PERM_USER
+SET ROLE :ROLE_DEFAULT_PERM_USER;
```

Comment:
Shouldn't we test both `SET ROLE` and `\c single :ROLE_DEFAULT_PERM_USER`

Reply:
Isn't that implicitly tested every time we do `create_hypertable()` in tests without `SET ROLE`? I think the interesting case is when `session_user` and `current_user` differs. I don't see why having `session_user` and `current_user` the same adds anything to the test that is not tested elsewhere pretty much every time we do `create_hypertable()`. Am I missing something?